beacon-chain/rpc/prysm/v1alpha1/validator/proposer_exits_test.go

```diff
@@ -28,11 +27,17 @@ func TestServer_getExits(t *testing.T) {
 	for i := primitives.ValidatorIndex(0); uint64(i) < params.BeaconConfig().MaxVoluntaryExits; i++ {
 		exit, err := util.GenerateVoluntaryExits(beaconState, privKeys[i], i)
 		require.NoError(t, err)
-		proposerServer.ExitPool.InsertVoluntaryExit(context.Background(), beaconState, exit)
+		proposerServer.ExitPool.InsertVoluntaryExit(exit)
 		exits[i] = exit
 	}
 
+	// the exit pool is a LIFO structure, so we expect exits in reverse order
```

Comment:
I designed the pool as LIFO for unity with the BLS pool, but happy to switch to FIFO.